### PR TITLE
feat(HACBS-2425): add jq and krb5-workstation to dockerfile

### PR DESCRIPTION
**Pull request by Troy876**
Update Dockerfile to include what's missing from the bundles Dockerfile.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,11 +6,12 @@ ARG OCP_VERSION=4.13.3
 ARG YQ_VERSION=4.34.1
 ARG GLAB_VERSION=1.31.0
 
-RUN curl -L https://github.com/mikefarah/yq/releases/download/v${YQ_VERSION}/yq_linux_amd64 -o /usr/bin/yq &&\
+RUN curl -L https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 -o /usr/bin/jq &&\
```
Comment (johnbieren) [inline Dockerfile:9]: Can you make the jq version an `ARG` like we have for the other tools?

**Review by johnbieren — CHANGES_REQUESTED** [review]
The commit isn't the proper format. It should be something like 
`feat(HACBS-2425): add jq and krb5-workstation to dockerfile`

File: Dockerfile (hunk 1 above)
Comment (Troy876) [reply Dockerfile:10]: ARG JQ_VERSION=1.6 has been added to the Dockerfile

Comment (johnbieren) [reply Dockerfile:10]: Can you use the variable in the url?

Comment (happybhati) [reply Dockerfile:10]: +1 Johnny
`RUN curl -L https://github.com/stedolan/jq/releases/download/jq-${JQ_VERSION}/jq-linux64 -o /usr/bin/jq &&\`

**Review by johnbieren — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (Troy876) [reply Dockerfile]: Should be updated now

**Pull request merged**